css-box-model/styles.css

```diff
@@ -0,0 +1,18 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin-left: 12px;
+  margin-right: 12px;
+  padding: 6px 16px;
+  border-radius: 6px;
+
+  /* margin-left: 12px;
```

Comment:
Please remove commented out code. 
Code comments are ok, such as line 5. But commented out code should always be removed :)

Reply:
<img width="362" alt="Screen Shot 2023-03-29 at 8 41 37 AM" src="https://user-images.githubusercontent.com/79718224/228593088-a2608ad5-f207-4912-98ed-5737f4c41078.png">